docs/controls/LayoutTransformControl.md

```diff
@@ -0,0 +1,70 @@
+---
+title: LayoutTransformControl
+author: odonno
+description: The LayoutTransformControl is a control that support transformations on FrameworkElement as if applied by LayoutTransform.
+keywords: windows 10, uwp, uwp community toolkit, uwp toolkit, LayoutTransformControl, RenderTransform, RotateTransform, ScaleTransform, SkewTransform 
+---
+
+# LayoutTransformControl
+
+The [LayoutTransformControl](https://docs.microsoft.com/en-us/dotnet/api/microsoft.toolkit.uwp.ui.controls.layouttransformcontrol) is a control that apply Matrix transformations on any `FrameworkElement` of your application.
```

Comment:
quick grammar fix here, should be "is a control that applies Matrix transformations"